README.md

```diff
@@ -7,7 +7,7 @@ Once a termination notice is received, it will try to gracefully stop all the po
 
 A helm chart has been created for this tool, and at time of writing was in the `incubator` repository.
```

Comment:
You might want to update this line as well.

Reply:
Done. Thanks.